crawler/src/main/resources/application.conf

```diff
@@ -0,0 +1,27 @@
+akka {
+  quartz {
+    defualtTimezone = "Europe/Warsaw"
+
+    schedules {
+      EverydayMidnight {
+        description = "Everyday at midnight"
+        expression = "0 0 0 * * ?"
+      }
+    }
+  }
+}
+
+bootstrap {
+  servers = "localhost:9092"
+}
+
+crawler {
+  http {
+    timeout = 10s
+    max.parallelism = 30
+  }
+}
+
+kafka {
+  topic = "newspaper-content"
+}
```

Comment:
I can live without it - in worse case I will add it myself later ;) 
But in general could it be useful - we won't wait till midnight to wait for pages to be fetched :P